OpenRA.Game/Actor.cs

```diff
@@ -320,6 +322,30 @@ public bool CanBeViewedByPlayer(Player player)
 			return defaultVisibility.IsVisible(this, player);
 		}
 
+		public IEnumerable<string> GetAllTargetTypes()
```

Comment:
Both seem reasonable to me to avoid overcluttering the `Actor` class. Any thoughts on where an extension method should be defined if I was going to do that?